circle.yml

```diff
@@ -9,12 +9,11 @@ dependencies:
   override:
     # Used by container_spec
     - docker pull alpine
-    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
-    - bundle install
+    - make image
 
 test:
   override:
-    - bundle exec rake
+    - make test
```

Comment:
Small consideration: I believe this will run `make image` *again*. Which should be fast since the image will already exist, but is doing needless work. Maybe we should just remove the build from `dependencies` and let it happen here?